### DISCUSSION TITLE
Quick fix unavailable in visual studio 2022

### DISCUSSION THREAD

**Discussion by navein97** [community/community#129036] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Quick fix unavailable in visual studio 2022. Im using latest vs 22 version

**Comment by ayim** [comment]
@navein97 can you clarify what you're expect to see from the quick fix?
Is it the lightbulb that's not showing?
Or are you looking for a Copilot feature that's not showing?

Either way, please use the "Report a Problem" feature in Visual Studio so that we can understand your environment and further diagnose.
![image](https://github.com/user-attachments/assets/a775d5f9-f3e7-47e3-9ec3-ed2f4251f810)

**Comment by navein97** [comment]
I'm looking for this feature on windows 10, latest visual studio 2022 (just
updated today). This picture was taken on macbook vs code Version: 1.90.1

[image: image.png]

**Reply by ayim** [reply]
Heyo, the image didn't attach - want to try again?

**Reply by navein97** [reply]
Sure! There yoou go
![image](https://github.com/community/community/assets/120290312/c0e332fd-6c7b-47fc-bc11-e8d10fba76fe)

**Reply by ayim** [reply]
@navein97 not all Copilot features are equivalent between products but we're working hard to make the experience more similar. I've added my coworker here for comment

@mikadumont is the expert on Fix with Copilot - is this feature something customers should expect to see, or is this something they should be looking for in a future build?

**Reply by mikadumont** [reply, marked as answer]
We plan to release Fix with Copilot in 17.11 GA. In the meantime, if you are using 17.11 P2 you can enable the experience in the Copilot Tools Options setting and select Enable "Fix with Copilot".
